compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/LangLibrary.java

```diff
@@ -96,27 +92,33 @@ public static LangLibrary getInstance(CompilerContext context) {
         return langLib;
     }
 
+    /**
+     * Returns the lang library method of the given type descriptor kind and the method name.
+     *
+     * @param type Type descriptor kind
+     * @param methodName Name of the lang library function
```

Comment:
We can align the javadoc comments for consistency and readability

```suggestion
     * @param type                Type descriptor kind
     * @param methodName Name of the lang library function
```